### ISSUE TITLE
Change stats colours for elite leagues

### ISSUE THREAD

**Issue by jordanrzymski** [jordanrzymski/statmaster#7]
We should collect manually on a google sheets the home and away colours for the elite leagues.... we can’t use white or grey and it conflicts with the background or no data 

For example England would be blue not white 

![image](https://user-images.githubusercontent.com/66678347/84346330-a7b88f80-aba7-11ea-86a1-88f27c1f861c.jpeg)

We might have to standize the colours and get the backend to check they don’t match so we output 2 different colours 

Elite lists only - start with the Germans bundersliga for testing

**Comment by scrabble85** [comment]
In hex?

**Comment by jordanrzymski** [comment]
Yeah are you able to do a compare function your side to make sure we don’t get 2 colours to same - like if away team equals home team colour change away team colour to their home colour

**Comment by scrabble85** [comment]
I may be able to get this information by doing a free trial on another package from the data provider

**Comment by scrabble85** [comment]
Yeah I can do  a if equals and return something else. I would need the hex you use currently so I can use that as a default

**Comment by jordanrzymski** [comment]
Happy to collect the data and set our colours so we know it’s done correctly and the fall back would be what we have now

**Comment by jordanrzymski** [comment]
default blue is #3880ff 
default yellow is #ffc409
